react/features/file-sharing/reducer.ts

```diff
@@ -20,7 +29,8 @@ ReducerRegistry.register<IFileSharingState>('features/file-sharing',
         newFiles.set(action.file.fileId, action.file);
 
         return {
-            files: newFiles
+            files: newFiles,
+            nbUnreadFiles: state.nbUnreadFiles
```

Comment:
Destructure state instead to avoid having to copy any other fields we might add later